src/raw/mod.rs

```diff
@@ -357,7 +357,8 @@ impl<T> RawTable<T> {
     pub fn new() -> Self {
         Self {
             data: NonNull::dangling(),
-            ctrl: NonNull::from(&Group::static_empty()[0]),
+            // Be careful to cast the entire slice to a raw pointer.
+            ctrl: unsafe { NonNull::new_unchecked(Group::static_empty().as_ptr() as *mut u8) },
```

Comment:
Btw, the `mut` cast here is actually meaningful -- the rest of the code must be careful not to write to this!  But I suppose that's already the case, this is a shared global static after all.